### DISCUSSION TITLE
Flicker of Unstyled Content (FOUC) with Gatsby and Material UI Plugin

### DISCUSSION THREAD

**Discussion by nabeelfarid** [gatsbyjs/gatsby#31255] [Help]
Site: https://gatsby-contentful-blogsite-nf.netlify.app/
Repo: https://github.com/nabeelfarid/gatsby-contentful-blogsite

Hi I am new to Gatsby and I can't seem to figure out how to get rid of the FOUC issues I am having. I am using the gatsby-plugin-material-ui plugin with gatsby 3 and MUI 4. I have looked at other similar issues posted and also looked at the gatsby plugin [examples](https://github.com/mui-org/material-ui/tree/master/examples/gatsby) but i am not sure what is it that I am missing or doing wrong. I would be grateful if you could please point me in the right direction.

Thanks

**Comment by zanedev** [comment]
I seem to always run into FOUC issues with MUI and gatsby it's really annoying. Usually the gatsby-material-ui plugin magically fixes it but I'm experiencing FOUC after an upgrade of MUI v4 to v5 with emotion on a site that was fine before and the plugin no longer fixes it :( Also https://github.com/hupe1980/gatsby-plugin-material-ui/issues/101
